server/util/jurisdiction_bulk_update.py

```diff
@@ -12,8 +12,8 @@
 ADMIN_EMAIL = "Admin Email"
 
 JURISDICTIONS_COLUMNS = [
-    CSVColumnType("Jurisdiction", CSVValueType.TEXT),
-    CSVColumnType("Admin Email", CSVValueType.EMAIL),
+    CSVColumnType("Jurisdiction", CSVValueType.TEXT, unique=True),
+    CSVColumnType("Admin Email", CSVValueType.EMAIL, unique=True),
```

Comment:
does this preclude a jurisdiction from having two admins? Or an admin for being an admin for two jurisdictions?

Reply:
No, those cases are both still allowed. I know it's not the best interface to express a composite unique key (I though it was a bit confusing when I originally wrote it), but it's been working so it didn't seem worth changing.